src/Controller/FeedController.php

```diff
@@ -31,10 +32,13 @@ public function index(DogCRUDService $dogService, PostRepository $postRepository
         
         $posts = $postRepository->findAll();
 
+        $comments = $commentRepository->findAll();
+
         return $this->render('feed/feed.html.twig', 
         [
             'posts' => $posts,
             'currentDogUser' => $postAuthor,
+            'comments' => $comments,
             ]
```

Comment:
Style and indenting looks a bit wonky here. Have a read of this https://www.php-fig.org/psr/psr-2/. I'd recommend doing this

```
        return $this->render('feed/feed.html.twig', [
            'posts' => $posts,
            'currentDogUser' => $postAuthor,
            'comments' => $comments,
        ]);

```

Reply:
I've fixed this in the PR branch feature/comments-CRUD.